lib/mzgl/app/mac/Metal/MZMetalView.mm

```diff
@@ -0,0 +1,109 @@
+#import "MZMetalView.h"
+#include "sokol_gfx.h"
+#include "EventDispatcher.h"
+#include "sokol_log.h"
+
+@implementation MZMetalView {
+	sg_pass_action pass_action;
+}
+
+static int sample_count				= 4;
+static sg_pixel_format depth_format = SG_PIXELFORMAT_NONE;
+
+- (id)initWithFrame:(NSRect)frame eventDispatcher:(std::shared_ptr<EventDispatcher>)evtDispatcher {
+	eventDispatcher = evtDispatcher;
+	self			= [super initWithFrame:frame device:MTLCreateSystemDefaultDevice()];
+	if (self != nil) {
+		self.delegate = self;
+		[self setSampleCount:(NSUInteger) sample_count];
+		[self setDevice:MTLCreateSystemDefaultDevice()];
+		self.preferredFramesPerSecond = 60;
+
+		self.colorPixelFormat = MTLPixelFormatBGRA8Unorm;
+		switch (depth_format) {
```

Comment:
In the default case, this is set to None, so is the depth format undefined then?